lib/OpenQA/CacheService/Task/Sync.pm

```diff
@@ -39,10 +39,16 @@ sub _cache_tests {
     my @cmd = (qw(rsync -avHP --timeout), RSYNC_TIMEOUT, "$from/", qw(--delete), "$to/tests/");
     my $cmd = join ' ', @cmd;
     $ctx->info("Calling: $cmd");
-    my $output = `@cmd`;
-    my $status = $? >> 8;
+    my $retry = 2;
+    while ($retry) {
+        $retry--;
+        my $output = `@cmd`;
+        my $status = $? >> 8;
+        $job->note(output => "[info] [#$job_id] Calling: $cmd\n$output");
+        last unless $status;
+        sleep 3;
+    }
```

Comment:
```suggestion
use constant RSYNC_RETRIES => $ENV{OPENQA_RSYNC_RETRIES} // 7;
use constant RSYNC_RETRY_PERIOD => $ENV{OPENQA_RSYNC_RETRY_PERIOD} // 3;
…
    for my $retry (1 .. RSYNC_RETRIES) {
        my $output = `@cmd`;
        my $status = $? >> 8;
        $job->note(output => "[info] [#$job_id] Calling try $retry: $cmd\n$output");
        last unless $status;
        sleep RSYNC_RETRY_PERIOD;
    }
```

Reply:
7 retries ?

Reply:
Yes, I always default to 7 retries as a magic number :) I am ok to keep 3 though, then just do the rest of the code suggestions

Reply:
j, usage of ENV vars is way better than some random numbers in code